tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -13,6 +13,7 @@ PACKAGES="chrony iptables dbus"
 [ "$MEASURED_ROOTFS" = yes ] && PACKAGES+=" cryptsetup-bin e2fsprogs"
 [ "$SECCOMP" = yes ] && PACKAGES+=" libseccomp2"
 REPO_URL=http://ports.ubuntu.com
+REPO_COMPONENTS=${REPO_COMPONENTS:-main}
```

Comment:
Just to clarify, one could simply set: REPO_COMPONENTS="main universe", correct?
I like the change, but we need to expose this so people can do:
`REPO_COMPONENTS="main universe" make rootfs-image-tarball`

Take a look at how we export the other vars used here so that can be done from that workflow as well, pelase.

Reply:
Done.